dashboard/app/controllers/openai_chat_controller.rb

```diff
@@ -8,6 +8,12 @@ def chat_completion
       return render(status: :bad_request, json: {})
     end
 
+    # Check for PII / Profanity
+    locale = params[:locale] || "en"
```

Comment:
Since this feature is English-only for now I feel ok just defaulting to English everywhere, but noting in my evolving future work doc that we'll want to address this later if we translate AI Tutor.